### ISSUE TITLE
Load all Playlist entries at once

### ISSUE THREAD

**Issue by blueYOSHI9000** [bundit/kord-app#60]
Currently, when clicking play on a song in a playlist Kord simply puts all the songs on-screen in the queue but not the rest. To put all songs into the queue I have to scroll all the way down until all songs are on-screen.
This also only seems to happen if a playlist just got synced, whether that's because it's a new playlist or because it got reloaded using the button at the top.

**Comment by bundit** [comment]
@blueYOSHI9000 Right, it currently paginates them when you reach the end of that list. Would it be helpful if it fetched the next set of tracks when you scroll to the bottom of the queue?

**Comment by blueYOSHI9000** [comment]
I don't think I quite follow but what I'd like to see (and completely forgot to include in the original post, sorry about that) is that all songs are loaded, even the ones that aren't on-screen. Basically making it so I no longer have to scroll down to include all songs.
Could fetch them the moment I click play, I don't mind having to wait a second for it to load if I no longer have to scroll down.
